src/components/slider/slider.tsx

```diff
@@ -150,6 +150,16 @@ export class Slider implements LabelableComponent, FormComponent, InteractiveCom
    */
   @Prop() scale: Scale = "m";
 
+  /**
+   * When true, displays numbers as comma separated (e.g. 1,970) and when false/not present does not (e.g. 1970)
+   */
+  @Prop({ reflect: true, mutable: true }) groupSeparator = false;
+
+  @Watch("groupSeparator")
```

Comment:
@Elijbet I don't think this watcher is necessary as it's just setting the value to what it has already been set as.